.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -6,19 +6,21 @@ body:
     attributes:
       value: |
         Thank you for taking the time to fill out this bug report. Please take the time to provide as much details as possible. Following our [bug report guidelines](https://devdocs.prestashop.com/8/contribute/contribute-reporting-issues/#best-practices-for-writing-an-issue) will help maintainers qualify your report faster.
+
+        Do not report bugs called "error 500" or "white page", this is a universal error and won't tell us anything. Instead, enable debug mode in Performance section of PrestaShop and try again. You will get a proper error message.
```

Comment:
```suggestion
        Reporting bugs with "error 500" or "white page" may not help us reproduce them. This is a universal error that hides what is underneath. Instead, enable debug mode in the Performance section of PrestaShop and try again. You should get a proper error message.
```

Reply:
@kpodemski `Reporting bugs with "error 500" or "white page" may not help us reproduce them.` - that's too soft. It won't tell us anything.

Reply:
I don't know which formulation we can use, but `Do not report bugs` is a negative message and it could be taken as being "aggressive"

Reply:
That's exactly why I decided to suggest a different form 👍🏻

Reply:
I know where are you heading, but I personally would go for some wording in the middle.

But, if you really want to go with `may not help us`, I won't argue. 👍

Reply:
I really want to go with a positive sentence 😊

Reply:
@Hlavtox we've seen a lot, sometimes we can know what's going on thanks for additional context provided in the report

> I've tested PrestaShop on PHP 8.2 and I have an error 500, see the screenshot

we know what's going on there :)